presto-spi/src/main/java/com/facebook/presto/spi/connector/ConnectorMetadata.java

```diff
@@ -230,6 +230,14 @@ default ConnectorPartitioningHandle getPartitioningHandleForExchange(ConnectorSe
         return listTables(session, schemaName.orElse(null));
     }
 
+    /**
+     * Gets the materialized views for a give table
+     */
+    default List<SchemaTableName> getMaterializedViews(ConnectorSession session, SchemaTableName tableName)
```

Comment:
- move this method closer to other materialized view interfaces
- the name will be confusing with `getMaterializedView`; maybe `getReferencedMaterializedViews`. Also, update the java doc to have more detailed explanation of the interface.